sys-apps/systemd/systemd-9999.ebuild

```diff
@@ -401,16 +401,16 @@ multilib_src_install_all() {
 		rmdir "${ED}${rootprefix}"/sbin || die
 	fi
 
-	if use hwdb; then
-		rm -r "${ED}${rootprefix}"/lib/udev/hwdb.d || die
-	fi
-
 	# Flatcar: Upstream uses keepdir commands to keep some empty
 	# directories.
 	#
 	# Flatcar: TODO: Consider using that instead of
 	# systemd_dotmpfilesd "${FILESDIR}"/systemd-flatcar.conf below.
 
+	if use hwdb; then
```

Comment:
Any reason for this change?

Reply:
I was using `vimdiff` for comparing the diff with upstream, and the `# Flatcar` comment seemed out of place.

Reply:
Alright.

I think this comment was about the stuff that exists in upstream ebuild, but we have dropped. So this comment is probably bound to seem out of place.